package.json

```diff
@@ -49,34 +54,69 @@
           "^.+\\.ts$": "ts-jest"
         },
         "testRegex": "(/__tests__/.*|(\\.|/)(test|spec))\\.(js|ts)$",
-        "moduleFileExtensions": ["ts", "js", "json"]
+        "moduleFileExtensions": [
+          "ts",
+          "js",
+          "json"
+        ]
       },
       {
         "displayName": "Prettier",
         "runner": "jest-runner-prettier",
-        "moduleFileExtensions": ["js", "ts", "md"],
-        "testMatch": ["src/**/*.js", "src/**/*.ts", "**/*.md"]
+        "moduleFileExtensions": [
+          "js",
+          "ts",
+          "md"
+        ],
+        "testMatch": [
+          "src/**/*.js",
+          "src/**/*.ts",
+          "**/*.md"
+        ]
       },
       {
         "displayName": "TsLint",
         "runner": "jest-runner-tslint",
-        "moduleFileExtensions": ["ts"],
-        "testMatch": ["**/*.ts"]
+        "moduleFileExtensions": [
+          "ts"
+        ],
+        "testMatch": [
+          "**/*.ts"
+        ]
       }
     ]
   },
+  "commitlint": {
+    "extends": [
+      "@commitlint/config-conventional"
+    ],
+    "rules": {
+      "scope-case": [
+        0
+      ]
+    }
+  },
+  "husky": {
```

Comment:
I think these should be merged into `scripts` 🤔 also it interferes with my PR #11 (will cause `precommit` to get called twice :()